software/standard-notes.yml

```
@@ -8,7 +8,7 @@ platforms:
 tags:
   - Note-taking & Editors
 source_code_url: https://github.com/standardnotes/app
-demo_url: https://app.standardnotes.org/
-stargazers_count: 5624
+demo_url: https://app.standardnotes.com/
+stargazers_count: 5622
```

```suggestion
demo_url: https://app.standardnotes.com/
stargazers_count: 5624
```

revert unrelated change